regtests/client/python/cli/command/catalogs.py

```diff
@@ -199,12 +194,33 @@ def execute(self, api: PolarisDefaultApi) -> None:
                     default_base_location=new_default_base_location,
                     additional_properties=new_additional_properties
                 )
-            if (self._has_aws_storage_info() or self._has_azure_storage_info() or self._has_gcs_storage_info() or
-                    self.allowed_locations or self.default_base_location):
+
+            if (self._has_aws_storage_info() or self._has_azure_storage_info() or
+                self._has_gcs_storage_info() or self.allowed_locations):
```

Comment:
`default_base_location` was removed here, intentional?

Reply:
Yes, I suspect it was originally present here because on the backend, changes to default_base_location do invoke the codepath for updating storage_config_info just to make sure default_base_location is present in the allowed_locations, but the client-side doesn't actually make any changes to storage_config_info when default_base_location is changed.